docs/backends/pybullet.rst

```diff
@@ -15,13 +15,13 @@ is based on a client-server architecture, there is no need to spin up any Docker
 containers to run the server.  This, along with its speed, may make PyBullet a
 preferable backend for COMPAS_FAB.  However, it, alone, does not provide motion
 planning functionality.  PyBullet is also not compatible with IronPython. Hence to use
-it with Rhinoceros and Grasshopper it must be invoked through the 
+it with Rhinoceros and Grasshopper it must be invoked through the
 :mod:`compas.rpc` module.
 
 Next Steps
 ==========
 
-* :ref:`Tutorial: COMPAS Robots <compas:robots>`
+* `Tutorial: COMPAS Robots <https://compas.dev/compas/1.17.9/tutorial/robots.html>`__
```

Comment:
Mmmm...its not so good to have a hard coded url to a specific compas version here, the previous syntax was nice because it relies on "intersphinx" to get the correct link everytime, but I guess this is broken now that robots moved to a separate package. Let's keep a TODO here and replace it with an intersphinx link to compas_robots as soon as that is ready

Reply:
Yea, I agree, its just we don't have a stable way to link to that Compas Docs yet.
Any idea how I can add a TODO in that doc string without messing up with the Sphinx parsing?
Its like a comment within a comment.

Reply:
@gonzalocasas

Reply:
it has a weird syntax with two dots:
```
..
  TODO: whatever
```

Reply:
Todo label added